### PR TITLE
June 2020 update

### LINKED ISSUE #22: SQL VM Resource Provider
Request from @nderhem  "We now have an option to register SQL VMs with Resource Provider which adds a lot of benefits to our customers. Is this something we can inject into this MCW?"

Reevaluate this at the next update.

### LINKED ISSUE #24: June 2020 Update Suggestions
Here are our suggested updates for the June 2020 update. Please add any other suggestions or feedback to this issue.

- Update diagrams to use new Azure icons
- Update to SQL 2019
- Add Key Vault to lab
- Add Always Encrypted with Secure Enclaves

### PR DESCRIPTION
**Pull request by paulopsgility**
Ready for SME review

Closes #22 
Closes #24

### CODE REVIEW

**Comment by sureshpaulraj** [comment]
**Task 2: Configure managed backup in SQL Server**
missing "**_@_**" enable_backup
image

**Comment by paulopsgility** [comment]
@sureshpaulraj this PR is for the june-2020-update. If you are testing this update you need to switch to the june-2020-update branch. You are looking at the master branch. The SQL managed backup task does not exist in the june-2020-update branch. We use the SQL vm resource provider for that now.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 1]
```diff
@@ -226,94 +226,93 @@ In this task, you will create the Azure Key Vault resource that will store and p
...
-    $vaultName = "<your key vault name>"
-    $appId = "<the application id you saved previously>"
+2. Select **Security** from the settings menu.
 
-    Set-AzKeyVaultAccessPolicy -VaultName $vaultName -ServicePrincipalName $appId -PermissionsToKeys get,list,wrapkey,unwrapkey
-    ```
+    ![](images/2020-06-18-18-05-23.png)
 
-### Task 6: Enable TDE 
+3. On the Security page, use the following configurations and select **Apply**.
```
Comment (sureshpaulraj) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:263]: On the Security page shows screen snapshot with more information including a mandatory field "SQL Connectivity"
image

Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:263]: That configuration is not necessary.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 2]
```diff
@@ -109,163 +116,280 @@ Cloud based disaster recovery site.
...
+    - Automated backup: **Enable**
+    - Retention period (days): **1**
+    - Storage account: ***Select the storage account you created in the previous task***
+    - Leave the other settings at the default settings. Note that you have options for encryption, backup of system databases and choosing either manual or automated backups. 
+
+    ![The SQL virtual machine backups configuration page with settings configured per the instructions.](images/2020-06-18-20-47-52.png "SQL virtual machine backup configuration.")
+
+## Exercise 3: Implement a Data Archive Strategy with SQL Server Stretch Database
 
 Duration: 30 minutes
```
Comment (sureshpaulraj) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:395]: ### Task 3: Implement Stretch DB based on date key 
8. Select the dropdown under Migration State. Notice that you have the option to pause the outbound migration.
Please note the option to pause the outbound is not a drop down under Migration State but,  a blue button at the top right hand corner with a label "Auto Refresh:"
image

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 3]
```diff
@@ -109,163 +113,287 @@ Cloud based disaster recovery site.
...
+
+8. On the review and create page, select **Create** to deploy your key vault.
+
+9.  After your Key Vault deployment completes, navigate to the resource, copy the **DNS Name** from the overview page and save it to Notepad for later use.
+
+    ![The Azure Key Vault overview page with the DNS name highlighted.](images/2020-06-17-18-46-02.png "Key vault overview.")
+
+10. Create a key by selecting **Keys** from the settings menu, then selecting the **+Generate/Import** button. Use the following options to create the key and select **Create**.
+
+    ![](images/2020-06-18-18-49-48.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:250]: Please add alt text.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 4]
```diff
@@ -109,163 +113,287 @@ Cloud based disaster recovery site.
...
+1. Navigate to the CloudShop resource group and open the **CloudShopSQL SQL virtual machine** resource.
 
-### Task 2: Configure managed backup in SQL Server
+    ![The CloudShopSQL Virtual machine and the CloudShopSQL SQL virtual machine resource are shown with the CloudShop SQL virtual machine resource highlighted.](images/2020-06-18-18-02-35.png "SQL virtual machine resource.")
 
-1. From within your SQL Server virtual machine, launch SQL Server Management Studio and connect to the local database instance, expand databases, and select the **AdventureWorks** database.
+2. Select **Security** from the settings menu.
 
-2. From within SQL Server Management Studio, select **New Query**.
+    ![](images/2020-06-18-18-05-23.png)
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:260]: Please add alt-text.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md [hunk 5]
```diff
@@ -109,163 +113,287 @@ Cloud based disaster recovery site.
...
+    - Automated backup: **Enable**
+    - Retention period (days): **1**
+    - Storage account: ***Select the storage account you created in the previous task***
+    - Leave the other settings at the default settings. Note that you have options for encryption, backup of system databases and choosing either manual or automated backups. 
+
+    ![The SQL virtual machine backups configuration page with settings configured per the instructions.](images/2020-06-18-20-47-52.png "SQL virtual machine backup configuration.")
+
+## Exercise 3: Implement a Data Archive Strategy with SQL Server Stretch Database
 
 Duration: 30 minutes
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:399]: Please review line 400 - it says "If you wish to implement table partitioning to archive data skip to Exercise 3"  (Line 400 is Exercise 3....should it be a different exercise?

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 2 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:399]: The auto refresh button controls the GUI updates for the current status, row counts etc. It does not actually control the migration state.

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 3 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:250]: Done

File: Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md (hunk 5 above)
Comment (paulopsgility) [reply Hands-on lab/HOL step-by step - SQL Server hybrid cloud.md:399]: Fixed

**Pull request merged**